server/api/auth.py

```diff
@@ -68,7 +81,7 @@ def signup():
                 "profile_image": "No image uploaded",
                 "favourite_recipe": "None yet,",
                 "favourite_cuisine": "None yet,",
-                "location": "Unknown Location"
+                "location": user.get_province_and_country
```

Comment:
Should this be city and province?

Reply:
I did this based on the mockups. If you look at the user profile you can see that it has "Toronto, Canada". I'm not sure how Canada names locales (meaning, is Toronto a province or city?) so I just used this thinking the province is toronto